scripts/install_third_party_libs.py

```diff
@@ -220,71 +219,63 @@ def ensure_system_python_libraries_are_installed(package, version):
 
 
 def rewrite_android_proto_files():
-    """Edit all android proto files."""
-    # Since there is no simple configuration for imports when using protobuf to
-    # generate Python files we need to manually fix the imports.
-    # See: https://github.com/protocolbuffers/protobuf/issues/1491
-    protobuf_dir = (
-        pathlib.Path(
-            os.path.join(
-                common.THIRD_PARTY_DIR,
-                'oppia-proto-api-8f3cde883c31785438e80656a5b6bb26bd01b6a1'))
-            .glob('**/*.proto'))
-    for p in protobuf_dir:
-        if p.suffix == '.proto':
+    """Android proto files are the part of oppia-proto-api-* directory in the
+    third_party_libs directory. Currently, proto files generation doesn't
+    support importing other proto files from a different directory.
+    Because of it, we had to keep all the proto files in the same directory.
+    Since there is no simple configuration for imports when using protobuf to
+    generate Python files we need to manually fix the imports.
+    See: https://github.com/protocolbuffers/protobuf/issues/1491
+    """
+    protobuf_dir = pathlib.Path(
+        common.THIRD_PARTY_OPPIA_ANDROID_PROTO_DIR).glob('**/*.proto')
+    for file in protobuf_dir:
```

Comment:
Is this a file (i.e. the file object containing the metadata, content, etc.) or a filename? Important to make sure the right word is used.

Reply:
it is a filename. Fixed.